.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+	"files.trimFinalNewlines": true,
```

Comment:
proofread pr

Reply:
After run yarn fix my code editor was adding new lines, that's why I have changed it. But for team, can it be irrelevant? If it is, I can delete it and only change my user settings.

Reply:
still needs to be removed even after adding .gitignore. also theres a typo in the gitignore

Reply:
Thanks a lot for your help Louis, I removed the file and fixed the typo.